azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java

```diff
@@ -64,6 +64,10 @@
     protected void doExecute() throws AzureExecutionException {
         // initialize library client
         az = getOrCreateAzureAppServiceClient();
+        if (az == null) {
+            getTelemetryProxy().trackEvent(INIT_FAILURE);
```

Comment:
Do we need to get exception from Account (if any)?

Reply:
Will fix in later pr